src/main/scala/firrtl/passes/memlib/MemUtils.scala

```diff
@@ -154,6 +155,10 @@ object createSubField {
   def apply(exp: Expression, n: String) = WSubField(exp, n, field_type(exp.tpe, n), UNKNOWNGENDER)
 }
 
+object createInstance {
```

Comment:
It would be better to put the helpers near their corresponding class declarations in WIR.scala. See WRef for an example.